tool/tctl/common/resource_command.go

```diff
@@ -407,13 +420,13 @@ func (rc *ResourceCommand) createRole(ctx context.Context, client auth.ClientI,
 		return trace.Wrap(err)
 	}
 	roleExists := (err == nil)
-	if roleExists && !rc.IsForced() {
+	if roleExists && !rc.force {
 		return trace.AlreadyExists("role '%s' already exists", roleName)
 	}
 	if err := client.UpsertRole(ctx, role); err != nil {
 		return trace.Wrap(err)
 	}
-	fmt.Printf("role '%s' has been %s\n", roleName, UpsertVerb(roleExists, rc.IsForced()))
+	fmt.Printf("role '%s' has been %s\n", roleName, UpsertVerb(roleExists, rc.force))
```

Comment:
nit: we're using `%q` in some places, and `'%s'` in other places. One results in `"foo"` and the other `'foo'`. If the point of this PR is to be more consistent, we should fix that :-)